apps/console/src/features/identity-providers/components/forms/authenticators/google-authenticator-form.tsx

```diff
@@ -175,24 +184,42 @@ export const GoogleAuthenticatorForm: FunctionComponent<GoogleAuthenticatorFormP
             const meta: CommonAuthenticatorFormFieldMetaInterface = metadata?.properties
                 .find((meta) => meta.key === value.key);
 
+            /**
+            * Parsing string  to boolean only for Google One Tap value
+            */
+            let localValue : any;
+
+            if(value.key === "IsGoogleOneTapEnabled" ) {
+                if( "true" === value.value ) {
```

Comment:
Flip condition and remove whitespace.
```suggestion
                if(value.value === "true") {
```

Reply:
Fixed 2fb29485d8588c2690eb42a8ca6460d9854a28f0